src/server/transaction.h

```diff
@@ -207,15 +207,14 @@ class Transaction {
 
   // Called by engine shard to execute a transaction hop.
   // txq_ooo is set to true if the transaction is running out of order
-  // not as the tx queue head.
-  // Returns true if the transaction continues running in the thread
+  // not as the tx queue head. Returns true if the transaction concludes.
```

Comment:
Had to check every `RunInShard` calls just to be sure the logic was inverted. Imo it's nicer now via the variable `concludes`